.circleci/config.yml

```diff
@@ -9,13 +9,13 @@ orbs:
 parameters:
   beta_build:
     type: boolean
-    default: false 
+    default: false
```

Comment:
Sorry for the noise here. You can use the "Hide whitespace changes" option to get rid of it in the diff.

<img width="627" alt="Screen Shot 2021-06-22 at 3 13 25 pm" src="https://user-images.githubusercontent.com/1218433/122867182-73f07800-d36c-11eb-87bf-0234505ded21.png">